frictionless/fields/number.py

```diff
@@ -64,6 +64,13 @@ def value_reader(cell):
                 Primary = float
                 Secondary = Decimal
             if isinstance(cell, str):
+                # for numeric string starting with zero for example, 001, 00, 01
+                cell_s = cell.strip()
+                has_leading_zero = (
+                    cell_s.startswith("0") and len(cell_s) > 1 and cell_s.isnumeric()
```

Comment:
I think we don't need isnumeric check here (if it's not not numeric it will fail later)